### ISSUE TITLE
cdc2Hs

### ISSUE THREAD

**Issue by Antonialock** [pombase/curation#2382]
there is an allele from pmid 8413179 called cdc2Hs(human cdc2 gene)

this sounds like a complementation experiment, do you remember Midori? You approved

**Comment by mah11** [comment]
ask Jacky - it's one of her sessions @JackyVH 

Also, this might fit in with some of the work @kimrutherford is doing for FlyBase - we could be getting close to having a way to do these in Canto because they need it.

**Comment by JackyVH** [comment]
Hi Both yes there is a cdc2Hs allele .  The pombe cdc2 gene was replaced with the cdc2 human gene which substituted for the pombe gene and allowed  cells to grow and divide .
Hope this helps

Jacky
Sent from my iPad

On 3 May 2019, at 13:01, Antonia <notifications@github.com<mailto:notifications@github.com>> wrote:


there is an allele from pmid 8413179 called cdc2Hs(human cdc2 gene)

this sounds like a complementation experiment, do you remember Midori? You approved

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fpombase%2Fcuration%2Fissues%2F2382&data=02%7C01%7C%7C555c952206304be4fd4108d6cfbf160c%7C4eed7807ebad415aa7a99170947f4eae%7C0%7C0%7C636924816950368160&sdata=R19KoAC5Ncr%2FH0oT0JbqpbNUZVNDHJdpNGkAOCYeuLo%3D&reserved=0>, or mute the thread<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FADKVK2AIMOPZXY7UILRDACDPTQSRZANCNFSM4HKTLSTA&data=02%7C01%7C%7C555c952206304be4fd4108d6cfbf160c%7C4eed7807ebad415aa7a99170947f4eae%7C0%7C0%7C636924816950368160&sdata=x2IsH2ZM9QPQv2vnok5oez9J2%2B%2FPLucZTVaJAptbRWo%3D&reserved=0>.

The Francis Crick Institute Limited is a registered charity in England and Wales no. 1140062 and a company registered in England and Wales no. 06885462, with its registered office at 1 Midland Road London NW1 1AT

**Comment by Antonialock** [comment]
Maybe we should discuss how to do these...I’ve always just done a complementation annotation

**Comment by ValWood** [comment]
WE can move this to complementation once we can capture the specific phenotypes complemented

**Comment by Antonialock** [comment]
By "waiting for pombase/canto#826 "

do you mean that you would like all our complementation data migrated to actual phenotype annotations (as opposed to simple classification e.g. partially complements, complements, doesn't complement) ? (if so, I wasn't aware)

**Comment by ValWood** [comment]
Yes longer term we would like to be able to specifiy which phenotypes are complemented. We discussed it at some point, possibly with flybase. It isn't a big deal for us as we don't have much complementation data that is really important. This one is somewhat a special case because these were the detailed asssays to  showed that human and pombe cdk1s were functionally equivalent. I think we should record this for historical purposes ;)

**Comment by kimrutherford** [comment]
> waiting for pombase/canto#826

#826 has been closed so now: pombase/canto#1915

**Comment by mah11** [comment]
tangent, but one more: 
S.p. dpm3 complements CHO Lec15 (cells don't express CHO dpm2; assay for transfected human CD59 product reaching cell surface - means GPI anchor restored)
PMID:10835346

**Comment by mah11** [comment]
another one:
cdc42-G12V suppresses localization of scd1 phenotype of gef1delta PMID:31719163 (left comment on annotation in canto session 721bf4355105bfb4)

**Comment by mah11** [comment]
PMID:9042863 34790a1dd257be4f cdc2-Y15F suppresses cell cycle arrest of chk1+ overexpression

**Comment by ValWood** [comment]
PMID:37160462
Phenotype complementation by human RAD23A
